obstacle_lcm_output.txt

```diff
@@ -0,0 +1,1768 @@
+----- 2020-10-16 00:46:03.445463
+{'bearing_deg': 6.469894763364437,
+ 'latitude_deg': 38,
+ 'latitude_min': 24.383276087527292,
+ 'longitude_deg': -110,
+ 'longitude_min': -47.51196922390136,
+ 'speed': -1.0}
```

Comment:
We shouldn't be saving txt files to our code repository, they can live in the google drive